pattern-categorization/innersource-program-mind-map.md

```diff
@@ -2,90 +2,106 @@
 
 ## Begin
 
-### [Start as an Experiment](https://github.com/InnerSourceCommons/InnerSourcePatterns/blob/master/patterns/2-structured/start-as-experiment.md)
+### Program Setup
 
-### [Dedicated Community Manager](https://github.com/InnerSourceCommons/InnerSourcePatterns/blob/master/patterns/2-structured/dedicated-community-leader.md)
+#### Management hesitates to invest in InnerSource
+
+##### [Start as an Experiment](https://patterns.innersourcecommons.org/p/start-as-experiment)
+
+#### Lack of credibility of the InnerSource program
+
+##### [Dedicated Community Leader](https://patterns.innersourcecommons.org/p/dedicated-community-leader)
 
 ### Project Setup
 
-#### [Standard Base Documentation](https://github.com/InnerSourceCommons/InnerSourcePatterns/blob/master/patterns/2-structured/project-setup/base-documentation.md)
+#### What is the project about and who maintains it
+
+##### [Standard Base Documentation](https://patterns.innersourcecommons.org/p/base-documentation)
 
-#### [Communication Tooling](https://github.com/InnerSourceCommons/InnerSourcePatterns/blob/master/patterns/2-structured/project-setup/communication-tooling.md)
+#### Getting help and contacting the project team
 
-#### [Issue Tracker Use Cases](https://github.com/InnerSourceCommons/InnerSourcePatterns/blob/master/patterns/2-structured/project-setup/issue-tracker.md)
+##### [Communication Tooling](https://patterns.innersourcecommons.org/p/communication-tooling)
+
+#### Intransparent roadmap and direction of the project
```

Comment:
How about something like `using issues for planning` for the issue tracker pattern? This would be consistent with the other two pattern descriptions showing what the setup is.

Reply:
This wouldn't be a description of a **problem** though, so with that inconsistent with the rest.
I was trying to describe an issue that can be addressed using this pattern.

Reply:
I see. Maybe making the other two project setup descriptions look more like a problem? Maybe something like `lack of maintainer communication` etc.?

Reply:
I have set some new problem descriptions and pushed them.